docs/development/virtual_environments.rst

```diff
@@ -195,18 +203,15 @@ Add your user to the kvm group to give it permission to run KVM:
    sudo modprobe kvm
    sudo modprobe kvm_intel
 
-
-Validate libvirt config
-^^^^^^^^^^^^^^^^^^^^^^^
-
-Log out, then log in again. Verify that libvirt is installed and KVM is
-available:
+Log out, then log in again. Verify that libvirt is installed:
 
 .. code:: sh
 
-   libvirtd --version
-   kvm-ok
+   sudo libvirtd --version
+
```

Comment:
A good one liner to check if virtualization support is ready could be:

`egrep -c '(vmx|svm)' /proc/cpuinfo`

This will show the number of lines that either have `vmx` (for Intel virtualization support) or `svm` (for AMD virtualization support). If the number is greater than 0, then support is enabled.

What do y'all think?

Reply:
Might want to make it `flags\s*:.* (vmx|svm) ` to tighten it down a bit.

Reply:
Gonna add that last one!